src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,4 +1,51 @@
 package techcourse.myblog.domain;
 
+
 public class Article {
-}
+    static final String DEFAULT_BACKGROUND = "/images/default/bg.jpg";
+    private String title;
+    private String contents;
+    private String background;
+
+    public Article(String title, String contents, String background) {
+        this.title = title;
+        this.contents = contents;
+        this.background = background;
+        if (this.background.isEmpty()) {
+            this.background = DEFAULT_BACKGROUND;
+        }
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요? setter 메서드는 도메인의 핵심 개념이나 의도를 코드에서 사라지게 해요.

```suggestion
    public void changeTitle(String title) {
```

Reply:
DTO를 만들면서 기존 Article 클래스의 필드들은 final로 선언하고 Setter들을 없앴습니다.